workflows/virology/pox-virus-amplicon/pox-virus-half-genome-test.yml

```diff
@@ -0,0 +1,46 @@
+- doc: Testing Pox Virus Illumina Amplicon Workflow with Lumpy Skin Disease Virus half genome sequenced data
+  job:
+    Reference FASTA:
+      class: File
+      location: "https://www.ebi.ac.uk/ena/browser/api/fasta/AF325528.1?download=true"
+      filetype: fasta
+    Primer Scheme (BED file with Pool identifiers):
```

Comment:
Can we put the instructions into the step annotations ?

I think this looks a bit nicer as `Primer Scheme`.
Same for `PE Pool1 read collection for samples` which could be `PE Pool1` and `PE Pool2 read collection for samples` which could be `PE Pool2`.

Reply:
Imagine we'll generate automated command line wrappers, much nicer to use `--primer-scheme`, `--pe-pool1` etc. I think that also looks better in the user interface